examples/nlp/language_modeling/conf/megatron_gpt_inference.yaml

```diff
@@ -9,7 +9,7 @@ inference:
   repetition_penalty: 1.2  # The parameter for repetition penalty. 1.0 means no penalty.
   min_tokens_to_generate: 0  # The minimum length of the sequence to be generated.
   compute_logprob: False  # a flag used to compute logprob of all the input text, a very special case of running inference, default False
-
+  end_strings: ["<|endoftext|>", "<extra_id_1>"]  # generation will stop when one of these tokens is generated
```

Comment:
I don't think we should add `<extra_id_1>` as part of the defaults here.

Reply:
Removed.